src/main/webapp/app/admin/audits/audit.model.ts

```diff
@@ -1,8 +1,6 @@
-import { AuditData } from './audit-data.model';
-
 export class Audit {
     constructor(
-        public data: AuditData,
+        public data: any,
```

Comment:
why not update AuditData?

Reply:
The audit data was already implemented as a `Map<String, Object>` on the back-end, so I'm not sure if there is a reason to maintain a class for it here. Perhaps instead of `any` a `Map<string, string>` is more appropriate here though.